tests/python_tests/test_llm_pipeline_static.py

```diff
@@ -119,20 +115,77 @@ def test_pipeline_from_blob(model_tmp_path, config, with_weights, model_id):
 
 
 @pytest.mark.precommit
-@pytest.mark.parametrize("config", pipeline_configs)
-@pytest.mark.parametrize("with_weights", blob_with_weights)
-@pytest.mark.parametrize("model_id", get_models_list())
-def test_pipeline_cache_dir(model_tmp_path, config, with_weights, model_id):
+@pytest.mark.parametrize(
+    "generation_config", 
+    [
+        pytest.param(get_greedy(), id="greedy"),
+        pytest.param(get_greedy_with_penalties(), id="greedy_with_penalties"),
+    ]
+)
+@pytest.mark.parametrize("llm_model", MODELS_LIST, indirect=True)
+@pytest.mark.parametrize("npu_config", PIPELINE_CONFIGS, indirect=True)
+@pytest.mark.xfail(reason="Generation result mismatch. Ticket 171117", raises=AssertionError)
+def test_generation_compare_with_stateful_list_input(
+    ov_model: LLMPipeline, 
+    npu_model: LLMPipeline,
+    generation_config: GenerationConfig,
+):
+    input_data = ['What is OpenVINO?']
+    ref_out = ov_model.generate(input_data, generation_config)
+    actual_out = npu_model.generate(input_data, generation_config)
+
+    assert ref_out == actual_out
+
+
+@pytest.mark.precommit
+@pytest.mark.parametrize(
+    "generation_config", 
+    [
+        pytest.param(get_greedy(), id="greedy"),
+        pytest.param(get_greedy_with_penalties(), id="greedy_with_penalties"),
+    ]
+)
+@pytest.mark.parametrize("llm_model", MODELS_LIST, indirect=True)
+@pytest.mark.parametrize("npu_config", PIPELINE_CONFIGS, indirect=True)
+@pytest.mark.xfail(reason="Generation result mismatch. Ticket 171117", raises=AssertionError)
+def test_generation_compare_with_stateful_chat_history(
+    llm_model: OVConvertedModelSchema,
+    npu_config: dict,
+    generation_config: GenerationConfig,
+):
+    # ChatHistory input sets internal chat state that conflicts with other input types
+    input_data = ChatHistory([{"role": "user", "content": 'What is OpenVINO?'}])
+    
+    ov_model_chat = LLMPipeline(llm_model.models_path, "CPU", **get_default_llm_properties())
+    ref_out = ov_model_chat.generate(input_data, generation_config)
+    
+    npu_model_chat = LLMPipeline(llm_model.models_path, "NPU", **npu_config)
+    actual_out = npu_model_chat.generate(input_data, generation_config)
+
+    assert ref_out == actual_out
```

Comment:
Outputs are instances of `DecodedResults` (as input is chat history object), result texts should be compared
```suggestion
    assert ref_out.texts == actual_out.texts
```